HealthcareSystem/PatientWebApp/wwwroot/js/patient_examinations.js

```diff
@@ -386,4 +403,90 @@ function cancelExamination(id) {
+                },
+                dataType: 'json',
+                processData: false,
+                contentType: false,
+                success: function (therapies) {
+
+                    let table = '<table style="margin-left:50px; margin-right:50px; margin-top:30px; margin-bottom:30px; width:300px;">'
+                        + '<tr> <td>Date:</td><td>' + report.dateAndTime.split('T')[0] + '</td> </tr>'
+                        + '<tr> <td>Time:</td><td>' + report.dateAndTime.split('T')[1] + '</td> </tr>'
+                        + '<tr> <td>Doctor:</td><td>' + report.doctorName + ' ' + report.doctorSurname + '</td> </tr>'
+                        + '<tr> <td>Anamnesis:</td><td>' + report.anamnesis + '</td> </tr>'
+                        + '<tr> <td>Prescripted therapies:</td><td>' + therapies.length + '</td> </tr>'
+                        + '</table>';
+
+                    $('#report_content').empty();
+                    $('#report_content').append(table);
+                    $('#bottomModalSuccess').modal('show');
+
+                },
+                error: function () {
+                    console.log("Error fetching therapies");
+                }
+            });
+        },
+        error: function () {
+            console.log("Error fetching report");
+        }
+    });
+};
+
+function showTherapies(examinationId) {
+
+    $('#topModalSuccess').modal('show');
+
+    $.ajax({
+        url: '/api/patient/examination/' + examinationId + "/therapies",
+        type: 'GET',
+        headers: {
+            'Authorization': 'Bearer ' + window.localStorage.getItem('token')
+        },
+        dataType: 'json',
+        processData: false,
+        contentType: false,
+        success: function (therapies) {
+            $('#therapy_content').empty();
+            if (therapies.length == 0) {
+                let messagge = '<label class="text-secondary" style="margin-left:24px; margin-top:30px; margin-bottom:30px">There are no prescribed therapies.</label>';
+                $('#therapy_content').append(messagge);
+            } else {
+                for (let i = 0; i < therapies.length; i++) {
+                    let therapy = '<table style="margin-left:50px; margin-right:50px; margin-top:30px; margin-bottom:30px; width:300px;">'
+                        + '<tr> <td>Start date:</td><td>' + therapies[i].startDate.split('T')[0] + '</td> </tr>'
+                        + '<tr> <td>End date:</td><td>' + therapies[i].endDate.split('T')[0] + '</td> </tr>'
+                        + '<tr> <td>Drug name:</td><td>' + therapies[i].drugName + '</td> </tr>'
+                        + '<tr> <td>Diagnosis:</td><td>' + therapies[i].diagnosis + '</td> </tr>'
+                        + '<tr> <td>Daily dose:</td><td>' + therapies[i].dailyDose + '</td> </tr>'
+                        + '</table>';
+                    $('#therapy_content').append(therapy);
```

Comment:
This table has no borders or anything, what will visually separate the therapies if there are multiple? Maybe just add a <code>\<hr\></code> between the tables or something.